site/public/index.php

```diff
@@ -65,10 +65,14 @@ function error_handler() {
 
 $semester = '';
 $course = '';
+$is_api = False;
 
 if ($core->getRouter()->hasNext()) {
     $first = $core->getRouter()->getNext();
-    if (in_array($first, ['authentication', 'home'])) {
+    if ($first === 'api') {
```

Comment:
Can api just be one of the components in the switch statement below?

Reply:
Yes, it can be one of the components. More modifications need to be added though because there are several operations like putting it into $_REQUEST['old']. But yes, we can.

In the meeting today we decided to replace the whole router with Symfony router. Hence it's likely that we are going to have an overhaul of index.php soon.